app/src/main/java/com/keylesspalace/tusky/util/LinkHelper.kt

```diff
@@ -131,7 +130,7 @@ private val trailingHashtagExpression by unsafeLazy {
 internal fun getTrailingHashtags(content: Spanned): Pair<Int, List<HashTag>> {
     // split() instead of lines() because we need to be able to account for the length of the removed delimiter
     val trailingContentLength = content.split('\r', '\n').asReversed().takeWhile { line ->
-        line.isBlank() || trailingHashtagExpression.matcher(line).matches()
+        line.split(whitespacePattern).all { it.isBlank() || hashtagWithHashPattern.matcher(it).matches() }
```

Comment:
Should this maybe be `splitToSequence`?

Reply:
yes, that should be slightly more efficient